sysdump/sysdump.go

```diff
@@ -1292,6 +1320,51 @@ func (c *Collector) submitFlavorSpecificTasks(ctx context.Context, f k8s.Flavor)
 	}
 }
 
+func (c *Collector) submitKVStoreTasks(ctx context.Context, pod *corev1.Pod) error {
+	for _, dump := range []struct {
+		name   string
+		prefix string
+	}{
+		{
+			name:   "identities",
+			prefix: "state/identities",
+		},
+		{
+			name:   "ips",
+			prefix: "state/ip",
+		},
+		{
+			name:   "nodes",
+			prefix: "state/nodes",
+		},
+		{
+			name:   "cnpstatuses",
+			prefix: "state/cnpstatuses",
+		},
+		{
+			name:   "heartbeat",
+			prefix: ".heartbeat",
+		},
+		{
+			name:   "services",
+			prefix: "state/services",
+		},
+	} {
+		stdout, stderr, err := c.Client.ExecInPodWithStderr(ctx, pod.Namespace, pod.Name, defaults.AgentContainerName, []string{
```

Comment:
I just committed a function, `ensureExecTarget`, which you may find useful here (albeit a bit overkill).

Reply:
Oh very nice, ill take a look at.